doc/syntax.org

```diff
@@ -0,0 +1,43 @@
+#+TITLE:   Markup Syntax Guide for Pancacke(d)
+#+AUTHOR:  Scott Wang
+#+STARTUP: align indent
+#+OPTIONS: H:5
+
+
+* Overview
```

Comment:
You can submit what you want to specify with this language as well :) that's what I meant by *collaborate*.